test/integration/cdi/example-driver/app/server.go

```diff
@@ -244,13 +245,32 @@ func NewCommand() *cobra.Command {
 	}
 	cmd.AddCommand(controller)
 
+	// kubelet plugin
+	kubeletPluginFlagSets := cliflag.NamedFlagSets{}
+	fs = kubeletPluginFlagSets.FlagSet("kubelet plugin")
+
+	kubeletDir := fs.String("kubelet-dir", "/var/lib/kubelet", "Path to the Kubelet directory.")
+	draAddress := fs.String("dra-address", *kubeletDir+"/plugins/"+*driverName+"/dra.sock", "Path to the DRA driver socket kubelet will use to issue CDI operations.")
```

Comment:
use [path.Join](https://pkg.go.dev/path#Join)

Reply:
Ah, yeah, I forgot to do for this part though I changed other parts. Thanks!